tgui-next/yarn.lock

```diff
@@ -0,0 +1,6231 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
should this file be in .gitignore or what

Reply:
No, because it's necessary for the package manager to behave deterministically, aka it precisely locks all dependency versions so that you always get predictable installs on any system, and it's not really updating that much.

Reply:
The true answer here is that YARN is being stupid and abusing the `.lock` extension in a way that creates confusion. To anybody unfamiliar with YARN or other equally stupid package managers, .lock is normally an extension used for temp per-session purposes, and not for long term storage.